rust/routee-compass-powertrain/src/routee/vehicle/default/bev.rs

```diff
@@ -46,12 +45,16 @@ impl VehicleType for BEV {
     fn name(&self) -> String {
         self.name.clone()
     }
+    fn state_variable_names(&self) -> Vec<String> {
+        vec![String::from("energy"), String::from("battery_state")]
```

Comment:
maybe we should be explicit here and call this `energy_electric` and remove the `energy` value from the default config?